modules/ROOT/pages/websocket/websocket-connector-messaging-operations.adoc

```diff
@@ -0,0 +1,68 @@
+= WebSocket Connection Management Components
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+== Send Operation
+
+The Send operation allows you to send a message through a WebSocket identified by its socket ID after a connection is established. This operation can be used for both inbound and outbound WebSocket instances.
+
+[source,xml,linenums]
+----
+    <flow name="sayHello">
+      <websocket:open-outbound-socket 
+        path="/chat/hello" 
+        config-ref="ws" 
+        target="socketId" 
+        targetValue="#[attributes.socketId]"/>
+      <websocket:send socketId="#[vars.socketId]" config-ref="ws">
+        <websocket:content>'Good morning!'</websocket:content>
+      </websocket:send>
+    </flow>
+
+    <flow name="sayHelloBack">
+        <websocket:inbound-listener 
+            path="/chat/hello" 
+            config-ref="ws" 
+            outputMimeType="text/plain" />
+        <websocket:send socketId="#[attributes.socketId]" config-ref="ws">
+            <websocket:content>#['Good morning ' ++ attributes.socketId ++ '!']</websocket:content>
+        </websocket:send>
+    </flow>
+----
+
+== Broadcast Operation
+
+The Broadcast operation sends a message to a group of sockets. To select the group of sockets to which the message can be broadcast:
+
+    * Subscribe to specific groups
+    * Connect to a specific path
+    * Be of a specific type (INBOUND or OUTBOUND)
+
+The type and group parameters work in tandem, in an additive way. They are both optional and can be used separately, but when used together they act as two levels of filtering:
+
+[source,xml,linenums]
+----
+<!-- Send to all sockets connected to an endpoint of path /quotes -->
+<websocket:broadcast path="/quotes" config-ref="ws" />
+
+<!-- Send to all inbound sockets connected to an endpoint of path /quotes -->
+<websocket:broadcast path="/quotes" type="INBOUND" config-ref="ws" />
+
+<!-- Send to all outbound sockets connected to the /quotes endpoint -->
+<websocket:broadcast path="/quotes" type="OUTBOUND" config-ref="ws" />
+
+<!-- Send to all INBOUND sockets connected to the /quotes endpoint, in the MARVEL and DC groups -->
+<websocket:broadcast path=”/quotes” type="INBOUND" groups=" #[['MARVEL', 'DC']]" config-ref="ws" />
+----
+
+=== Output
```

Comment:
Per Mariano G, this is incorrect. 
"The operation is void which is completely different than this."